value.go

```diff
@@ -215,14 +215,14 @@ func (r *Value) nth(column int) zcode.Bytes {
 	return zv
 }
 
-func (r *Value) Columns() []Column {
-	return TypeRecordOf(r.Type).Columns
+func (r *Value) Fields() []Field {
+	return TypeRecordOf(r.Type).Fields
 }
 
 func (v *Value) DerefByColumn(col int) *Value {
```

Comment:
Should this be renamed `DerefByField`?

Reply:
Indeed.

Reply:
On second thought, I think this should become DerefByIndex or FieldByIndex, so I'll leave it as is and we'll sort out the name in another PR.